whylog/config/parser_name_manager.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+
+class ParserNameManager(object):
+    WORDS_COUNT_IN_NAME = 2
+
+    def __init__(self, parsers):
+        self._parsers_name = set()
```

Comment:
AbstractConfig already has a structure in which you can efficiently check for presence of a parser name. Use that instead of creating a duplicate structure and keeping it in sync with the existing one.